app/javascript/packs/application.js

```diff
@@ -7,10 +7,9 @@
 // To reference this file, add <%= javascript_pack_tag 'application' %> to the appropriate
 // layout file, like app/views/layouts/application.html.erb
 
-import 'styles/application.scss'
-
 import React from 'react';
 import ReactDOM from 'react-dom';
```

Comment:
was there a reason you reordered this import?   we should figure out a standard import ordering scheme